mkosi/__init__.py

```diff
@@ -5368,6 +5383,20 @@ def create_parser() -> ArgumentParserMkosi:
         default=0,
         help="Wait up to SECONDS seconds for the SSH connection to be available when using 'mkosi ssh'",
     )
+    group.add_argument(
+        "--ssh-agent",
+        type=parse_ssh_agent,
+        default=False,
```

Comment:
The default should then be `None`
```suggestion
        default=None,
```